kafka-connect-temp/src/main/scala/com/datamountaineer/streamreactor/temp/KcqlSettings.scala

```diff
@@ -21,4 +21,16 @@ trait KcqlSettings extends BaseSettings {
   def getIgnoreFields(routes: Set[Config] = getRoutes): Map[String, Set[String]] = {
     routes.map(rm => (rm.getSource, rm.getIgnoredField.toSet)).toMap
   }
+
+  def getPrimaryKeys(routes: Set[Config] = getRoutes): Map[String, Set[String]] = {
+    routes.map(r => (r.getSource, r.getPrimaryKeys.toSet)).toMap
```

Comment:
If I may be picky, why the use of 'r' variable in some places, and 'rm' in another? 
- no need to actually extend this variable name for this scenario, just something in terms of consistency - pick one? =)